docs/uploadAemAsset.html

```diff
@@ -0,0 +1,55 @@
+---
+layout: page
+title: AEM asset upload API
+---
+
+<p>Campaign API for downloading & uploading(publishing) AEM cloud asset on campaign publication server. Just like all APIs in the SDK, it's been wrapped into a function and will return a JSON object containing publishedUrl.</p>
+<p class="warning">Note: This API is restricted to be used from acc-web-ui only. It is used when an asset is selected from
```

Comment:
I do not like this very much? The SDK is supposed to be a general purpose SDK and should not depend on ACC Web UI or any particular usage. Why is there this restriction?

Reply:
Hi, The API isn't restricting us to be only called from web-ui, but i can't find any usecase of it other than web-ui.
Code added is for feature which shows aem cloud or asset essentials asset library within email-designer acrite.
This API gets an assetInfo url instead of direct download url from acrite(acrite gets it from https://git.corp.adobe.com/CQ/assets-selectors)- assetInfo url when called with GET method & auth token returns a short lived signed url for downloading the asset.